### ISSUE TITLE
add topics

### ISSUE THREAD

**Issue by Beliavsky** [ITNG/ParizPLOS2021#2]
I suggest adding topics such as `computational-biology`, `neuroscience` in the About section at https://github.com/ITNG/ParizPLOS2021

**Comment by Ziaeemehr** [comment]
Hi  :wave: 
@arefpz

**Comment by arefpz** [comment]
Thanks. Right, I should put more comments in the About section. Hopefully, soon!


> I suggest adding topics such as `computational-biology`, `neuroscience` in the About section at https://github.com/ITNG/ParizPLOS2021

**Comment by Beliavsky** [comment]
No, my suggestion was to add topic tags that can be searched. Please see https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/classifying-your-repository-with-topics .
